src/test/scripts/org/reaktivity/k3po/nukleus/ext/simplex/server.sent.throttle.initial.only.update.none/server.rpt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+accept "nukleus://server/streams/client"
+       option nukleus:route 1234L
+       option nukleus:transmission "simplex"
+       option nukleus:window 1
+       option nukleus:update "none"
+
+accepted
+connected
+
+read "Hello, world"
```

Comment:
This test should be failing with timeout but it's not (because the Travis build past, as David pointed out). So we have about the fix. However this test should be written differently to verify the expected behavior, since relying on the timeout is not a good method (slows up the build and it's not precise). What I would suggest you do is as follows (instead of read "Hello, world"), combined with the changes I suggested in my comment on the other script:
```
read "H"
read notify FIRST_BYTE_RECEIVED
```
At this point the other script will send a second byte which should cause this site to automatically send a reset because it violates the total we know we gave the client (still only 1 because of the update:none option)

Reply:
I've updated it.